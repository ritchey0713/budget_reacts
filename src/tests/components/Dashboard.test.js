import React from 'react'
import { shallow } from "enzyme"

import Dashboard from "../../components/DashBoard"

test("Should render the dashboard component", () => {
  const wrapper = shallow(<Dashboard />)
  expect(wrapper).toMatchSnapshot()
})