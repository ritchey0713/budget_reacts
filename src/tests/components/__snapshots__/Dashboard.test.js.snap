// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the dashboard component 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
